pkg/cmd/pulumi/new.go

```diff
@@ -572,6 +573,33 @@ func errorIfNotEmptyDirectory(path string) error {
 
 func validateProjectName(ctx context.Context, b backend.Backend,
 	orgName string, projectName string, generateOnly bool, opts display.Options,
+) error {
+	handleExistingProjectName := func(projectName string) error {
+		prompt := "\b\n" + opts.Color.Colorize(colors.SpecPrompt+
+			fmt.Sprintf("A project with the name `%s` already exists.\n", projectName)+
```

Comment:
Elsewhere we don't have the newline before the prompt

```suggestion
			fmt.Sprintf("A project with the name `%s` already exists.", projectName)+
```